### ISSUE TITLE
log4j:ERROR Could not create an Appender

### ISSUE THREAD

**Issue by HiromuHota** [HiromuHota/pentaho-kettle#129]
## Expected Behavior

No error message.

## Actual Behavior

```
log4j:ERROR Could not create an Appender. Reported error follows.
java.lang.ClassCastException: org.apache.log4j.ConsoleAppender cannot be cast to org.apache.log4j.Appender
	at org.apache.log4j.xml.DOMConfigurator.parseAppender(DOMConfigurator.java:248)
	at org.apache.log4j.xml.DOMConfigurator.findAppenderByName(DOMConfigurator.java:176)
	at org.apache.log4j.xml.DOMConfigurator.findAppenderByReference(DOMConfigurator.java:191)
	at org.apache.log4j.xml.DOMConfigurator.parseChildrenOfLoggerElement(DOMConfigurator.java:523)
	at org.apache.log4j.xml.DOMConfigurator.parseCategory(DOMConfigurator.java:436)
	at org.apache.log4j.xml.DOMConfigurator.parse(DOMConfigurator.java:1004)
	at org.apache.log4j.xml.DOMConfigurator.doConfigure(DOMConfigurator.java:872)
	at org.apache.log4j.xml.DOMConfigurator.doConfigure(DOMConfigurator.java:755)
	at org.apache.log4j.xml.DOMConfigurator.configure(DOMConfigurator.java:896)
	at org.pentaho.di.core.logging.log4j.Log4jLogging.applyLog4jConfiguration(Log4jLogging.java:101)
	at org.pentaho.di.core.logging.log4j.Log4jLogging.createLogger(Log4jLogging.java:109)
	at org.pentaho.di.core.logging.log4j.Log4jLogging.init(Log4jLogging.java:71)
	at org.pentaho.di.core.KettleClientEnvironment.initLogginPlugins(KettleClientEnvironment.java:176)
	at org.pentaho.di.core.KettleClientEnvironment.init(KettleClientEnvironment.java:120)
	at org.pentaho.di.core.KettleClientEnvironment.init(KettleClientEnvironment.java:80)
	at org.pentaho.di.core.KettleEnvironment.init(KettleEnvironment.java:133)
	at org.pentaho.di.core.KettleEnvironment.init(KettleEnvironment.java:100)
	at org.pentaho.di.ui.spoon.WebSpoonServletContextListener$1.call(WebSpoonServletContextListener.java:79)
	at org.pentaho.di.ui.spoon.WebSpoonServletContextListener$1.call(WebSpoonServletContextListener.java:71)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
log4j:ERROR Could not create an Appender. Reported error follows.
java.lang.ClassCastException: org.apache.log4j.ConsoleAppender cannot be cast to org.apache.log4j.Appender
	at org.apache.log4j.xml.DOMConfigurator.parseAppender(DOMConfigurator.java:248)
	at org.apache.log4j.xml.DOMConfigurator.findAppenderByName(DOMConfigurator.java:176)
	at org.apache.log4j.xml.DOMConfigurator.findAppenderByReference(DOMConfigurator.java:191)
	at org.apache.log4j.xml.DOMConfigurator.parseChildrenOfLoggerElement(DOMConfigurator.java:523)
	at org.apache.log4j.xml.DOMConfigurator.parseRoot(DOMConfigurator.java:492)
	at org.apache.log4j.xml.DOMConfigurator.parse(DOMConfigurator.java:1006)
	at org.apache.log4j.xml.DOMConfigurator.doConfigure(DOMConfigurator.java:872)
	at org.apache.log4j.xml.DOMConfigurator.doConfigure(DOMConfigurator.java:755)
	at org.apache.log4j.xml.DOMConfigurator.configure(DOMConfigurator.java:896)
	at org.pentaho.di.core.logging.log4j.Log4jLogging.applyLog4jConfiguration(Log4jLogging.java:101)
	at org.pentaho.di.core.logging.log4j.Log4jLogging.createLogger(Log4jLogging.java:109)
	at org.pentaho.di.core.logging.log4j.Log4jLogging.init(Log4jLogging.java:71)
	at org.pentaho.di.core.KettleClientEnvironment.initLogginPlugins(KettleClientEnvironment.java:176)
	at org.pentaho.di.core.KettleClientEnvironment.init(KettleClientEnvironment.java:120)
	at org.pentaho.di.core.KettleClientEnvironment.init(KettleClientEnvironment.java:80)
	at org.pentaho.di.core.KettleEnvironment.init(KettleEnvironment.java:133)
	at org.pentaho.di.core.KettleEnvironment.init(KettleEnvironment.java:100)
	at org.pentaho.di.ui.spoon.WebSpoonServletContextListener$1.call(WebSpoonServletContextListener.java:79)
	at org.pentaho.di.ui.spoon.WebSpoonServletContextListener$1.call(WebSpoonServletContextListener.java:71)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

## Steps to Reproduce the Problem

  1.  `docker run -d -p 8080:8080 -name webspoon hiromuhota/webspoon:nightly-full`
  1.  `docker logs -f webspoon`

### Server
  - webSpoon (e.g., version: 0.8.0.13 or commitId: 0211d84b): 7a1652ba

**Comment by HiromuHota** [comment]
[PDI-14754](https://jira.pentaho.com/browse/PDI-14754) might be related.

**Comment by HiromuHota** [comment]
Just realized that this issue disappeared as of fd6accbbf94d0200891d4a687a3e03da79dbfca7.
Probably this was resolved by 728c046c3e03680061543d9ae92ccdd585184fe2.
